<?xml version="1.0"?>
<launch>

  <!-- start twist to ackermannDrive converter for robot steering plugin -->
  <node pkg="sim_simple_controller" name="twist_to_ackermannDrive" type="twist_to_ackermannDrive.py" args="twistToAckermannDrive:=eufsa_control/teleop_vel_out" />
  
  <!-- start rqt robot steering module -->
  <node pkg="rqt_robot_steering" name="rqt_robot_steering" type="rqt_robot_steering" />
  
  <!-- start rqt robot steering module -->
  <!-- <node pkg="rqt_robot_steering" name="rqt_robot_steering" type="rqt_robot_steering" /> -->
  <node name="sim_simple_controller" pkg="sim_simple_controller" type="sim_simple_controller" output="screen">
    <remap from="joint_states" to="eufs/joint_states"/>
    <param name="model" value="eufs"/>
    <param name="publish_odom_tf" value="true"/>
    <param name="wheelbase" value="1.55"/>
    <param name="wheel_diameter" value="0.505"/>
    <param name="max_speed" value="10.0"/>
    <param name="max_steering" value="0.523599"/> <!-- 30 degs -->
  </node> 
  
</launch>
